src/config.ts

```diff
@@ -197,7 +209,7 @@ function parseConfigFile(configData: string): ConfigurationOptionsPartial {
     }
     return data
   } catch (error) {
-    throw error
+    throw new Error(`error validating config: ${error}`)
```

Comment:
```suggestion
    throw new Error(`error validating config: ${(error as Error).message}`)
```

Worth casting to get the message here? Not sure what the output would be if this is an object.

Reply:
The reason to cast is to please the TS type checker, because `error` is only defined inside the `catch` we need to give it some hints so it knows it can call `message` to get the error body. I'm avoiding

Reply:
☝️ I think what Mitchell is pointing out is that we're missing the `message` call after the case there?